daffodil-cli/LICENSE-bin

```diff
@@ -261,14 +273,14 @@ subcomponents is subject to the terms and conditions of the following licenses.
     ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
     POSSIBILITY OF SUCH DAMAGE.
 
-  This product bundles material copied or derived from W3C. Specific files
-  include:
-    - XMLSchema.dtd (https://www.w3.org/2001/XMLSchema.dtd)
-    - XMLSchema.xsd (https://www.w3.org/2001/XMLSchema.xsd)
-    - xml.xsd (https://www.w3.org/2001/xml.xsd)
-    - datatypes.dtd (https://www.w3.org/2001/datatypes.dtd)
-    - XMLSchema_for_DFDL.xsd (https://www.w3.org/2001/XMLSchema.dtd)
-  The source is available under the following license:
+  This product bundles material copied or derived from W3C, including the
```

Comment:
Same here.  If this path exists in the binary great, but more likely than not it won't.

Reply:
Check, will fix.